.github/CODEOWNERS

```
@@ -40,6 +40,9 @@
 # verification team
 /roles/jenkins_job_launcher          @redhatci/verification
 /roles/junit2json                    @redhatci/verification
+/plugins/filter                      @redhatci/verification @redhatci/dci
```

No need to request dci to check for filters owned by verification:
```suggestion
/plugins/filter/junit2obj.py         @redhatci/verification
```